alts/worker/runners/base.py

```diff
@@ -488,11 +500,7 @@ def run_third_party_tests(self):
                         docker_args=['--workdir', workdir],
                         workdir=workdir,
                         artifacts_key=f'third_party_test_{file.name}',
-                        additional_section_name=(
-                            TESTS_SECTION_NAME
-                            if isinstance(executor, BatsExecutor)
-                            else THIRD_PARTY_SECTION_NAME
-                        ),
+                        additional_section_name=THIRD_PARTY_SECTION_NAME,
```

Comment:
does it work for integrity tests? as far as I remember, I did that for correct parsing tap logs on the web server side